### PR TITLE
Fixed critical segfault, memory leak and default keepalive

### PR DESCRIPTION
**Pull request by argakon**

### CODE REVIEW

File: mosquitto.c [hunk 1]
```diff
@@ -1147,7 +1147,12 @@ PHP_MOSQUITTO_API void php_mosquitto_message_callback(struct mosquitto *mosq, vo
 
 	object_init_ex(message_zval, mosquitto_ce_message);
 	message_object = mosquitto_message_object_from_zend_object(Z_OBJ_P(message_zval));
-	mosquitto_message_copy(&message_object->message, message);
+	message_object->message.mid = message->mid;
```
Comment (mgdm) [inline mosquitto.c]: This caused a leak? Was it logged by PHP's memory leak detection or was it something you found in Valgrind or similar?

Comment (argakon) [inline mosquitto.c]: Yes.
At first i used 2 simple scripts. One for subscribing and second for publishing 100000 messages. I run this scripts and after that watch in htop linux utility at memory column. In htop memory always grow.
After that i run subscribe script in valgrind, i saw the problem. Leak was in PHP5 and PHP7

Comment (mgdm) [inline mosquitto.c]: Ahh, yes, I see now that I've looked at how that function is implemented: 
https://github.com/eclipse/mosquitto/blob/master/lib/messages_mosq.c#L59

Good catch, thanks!

File: mosquitto.c [hunk 2]
```diff
@@ -1147,7 +1147,12 @@ PHP_MOSQUITTO_API void php_mosquitto_message_callback(struct mosquitto *mosq, vo
...
+	message_object->message.qos = message->qos;
+	message_object->message.retain = message->retain;
+	message_object->message.topic = message->topic;
+	message_object->message.payload = message->payload;
```
Comment (argakon) [inline mosquitto.c]: As you wish. I'am not a C developer, my commit is a just quick fix.

**Pull request merged**